dbbackup/storage/ftp_storage.py

```diff
@@ -44,6 +44,7 @@ def _check_settings(self):
     #  DBBackup Storage Methods
     ###################################
 
+    @property
```

Comment:
If you add a decorator, you would need to check all calls to the method and fix them, too?